salt/journal-cms/api-dummy.sls

```diff
@@ -0,0 +1,9 @@
+api-dummy-nginx-vhost-dev:
```

Comment:
this file needs to be considered locally, along with the original api-dummy files it depends upon (that come from `api-dummy-formula`). Add this entry https://github.com/elifesciences/search-formula/blob/master/salt/example.top#L17 and this entry https://github.com/elifesciences/search-formula/blob/master/salt/example.top#L20 to the `example.top` file of this repository; it will be used locally to put together all the .sls file to create a Vagrant VM